proxygen/httpserver/samples/echo/EchoServer.cpp

```diff
@@ -30,9 +33,12 @@ DEFINE_int32(http_port, 11000, "Port to listen on with HTTP protocol");
 DEFINE_int32(spdy_port, 11001, "Port to listen on with SPDY protocol");
 DEFINE_int32(h2_port, 11002, "Port to listen on with HTTP/2 protocol");
 DEFINE_string(ip, "localhost", "IP/Hostname to bind to");
-DEFINE_int32(threads, 0, "Number of threads to listen on. Numbers <= 0 "
+DEFINE_int32(threads, 1, "Number of threads to listen on. Numbers <= 0 "
              "will use the number of cores on this machine.");
 
+std::atomic<int> numThreads;
```

Comment:
Making these global is not really ideal - would prefer they be part of something else.